integration-commons/src/test/java/org/symphonyoss/integration/exception/IntegrationRuntimeExceptionTest.java

```diff
@@ -0,0 +1,76 @@
+package org.symphonyoss.integration.exception;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link IntegrationRuntimeException}.
+ * Created by crepache on 23/06/17.
+ */
+public class IntegrationRuntimeExceptionTest {
+
+  private String message = "message";
+
+  private String component = "component";
+
+  private String solution = "solution";
+
+  private Throwable cause = new Throwable("cause");
+
+  @Test
+  public void testIntegrationRuntimeException() {
+    IntegrationRuntimeException exception = new IntegrationRuntimeException(component, message);
+    String resultMessage = exception.getMessage();
+    String expectedMessage = "\n" +
+        "Component: component\n" +
```

Comment:
You could create a private method to build these String messages and avoid repeated code.